packages/app/src/pages/transaction/index.tsx

```diff
@@ -95,6 +100,13 @@ export const Transaction: React.FC = () => {
   };
 
   const setupWithState = async (tx: TransactionPayload) => {
+    if (tx.network) {
+      const network =
+        tx.network.version == TransactionVersion.Mainnet
```

Comment:
Can use strict `===` here

Reply:
addressed in https://github.com/blockstack/ux/pull/621/commits/8e85e35c79e8cb9be95a1606aece2720fc9e454d